Fix bug when logging outside of hangfire.

SDEBEUL opened the pull request:
https://github.com/AnderssonPeter/Hangfire.Console.Extensions/issues/1
Fix bug for serialization to seq.
https://discuss.hangfire.io/t/serilog-seq-with-hangfire-console/7428/2

AnderssonPeter commented:
A few questions.
- what is the point of `PerformContextValueXX`?
- why are you passing null to `PerformContextValue`? (im not home so I can't check what it does)

SDEBEUL commented:
Peter,

The "PerformContextValueXX" was my mistake. That was your original class that I renamed. 
I removed it.

The LogEventPropertyValue was switched to  ScalarValue.
and I took that from [this](https://discuss.hangfire.io/t/serilog-seq-with-hangfire-console/7428/2) example.

Br

AnderssonPeter reviewed (commented):
Only enrich with hangfire job when it exists.
Add as scalar value
